### ISSUE TITLE
rails naming conventions and the ons_constituencies table

### ISSUE THREAD

**Issue by baob** [swapmyvote/swapmyvote#490]
Relates to #459

> I can see there are technical problems to be resolved, but also naming problems. I've referred to to the strings as 'ONS ids' whereas they seem to be designated as 'constituency codes' or a 'PCONCD' by the ONS open geography portal

> if we adopted 'ons_code' or 'pconcd' or 'pcon_code' for the string, that would free us up to use 'constituency_ons_id' according to the rails conventions, as the id of the table ons_constituencies.

> aspiers commented yesterday
> @baob Please can you file a separate issue for that if you didn't already?

**Comment by aspiers** [comment]
@baob I'm a little unclear on precisely what this issue is for; please can you add a verb to the title to make it actionable?

Also worth noting that if we can merge #456 then we could rename the `ons_constituencies` model to `constituencies` which would give us a bit more flexibility.

**Comment by baob** [comment]
This will be unblocked by the merge of #654

**Comment by baob** [comment]
> This will be unblocked by the merge of #654

Which is now done. This issue can be actioned.
